src/components/app.jsx

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
+import {BrowserRouter as Router, Route} from 'react-router-dom';
 import TestComponent from './testComponent';
+import Home from './home';
 
 import classNames from 'classnames/bind';
 import Styles from './styles.scss';
-const cx = classNames.bind(Styles)
+const cx = classNames.bind(Styles);
 
 const App = () => (
-  <div className={cx('app')}>
-    <h1>Hello, emilio</h1>
-    <TestComponent hello='world'/>
-  </div>
+  <Router>
```

Comment:
Router shouldn't include visual styles in it, and this is no longer a component. This can be restructure to better separate concerns.